ui/guest/src/controls/rte.ts

```diff
@@ -177,7 +179,8 @@ export function initTinyMCE(
         )
         .subscribe(({ payload }) => {
           if (payload) {
-            cb(payload.path, { alt: payload.name });
+            const path = meta.filetype === 'file' ? getPreviewURLFromPath(payload.path) : payload.path;
```

Comment:
Why alter the path for "files"?

Reply:
File type may be 'image', 'media' or 'file'. 'image' and 'media' are the ones used when adding images or videos. File type is the one returned when selecting on the link picker.
When selecting a link (using the file browse datasource) we get an absolute path, and just like in legacy RTE, we need to return the previewUrl.